src/components/geonode.jsx

```diff
@@ -106,6 +106,21 @@ class GeoNodeViewer extends React.Component {
       errorOpen: false
     });
   }
+  _createLayerList() {
+    let layerList;
+    if(this._local) {
+      layerList = {
+        sources: [{title: this._local.title, url: this._local.url, type: 'WMS'}],
+        allowUserInput: true
+      };
+    } else {
+      layerList = {
+        sources: [{title: 'Local Geoserver', url: this.props.server+'/geoserver/wms', type: 'WMS'}],
```

Comment:
yes, maybe we change that to add a geoserver url... not sure, need to think about it